build-system/compile/bundles.config.js

```diff
@@ -16,6 +16,7 @@
 'use strict';
 
 const argv = require('minimist')(process.argv.slice(2));
+/** @type {Array<Object>} */
```

Comment:
It seems weird to arbitrarily add a type annotation to a `require` statement. Where is this error originating from? Is there a way to insert this type info inside `bundles.config.extensions.json` so it doesn't have to be included here?

Reply:
I've managed to fix the type error stemming from this!